### ISSUE TITLE
Add support for incremental rsync

### ISSUE THREAD

**Issue by aelkiss** [hathitrust/datasets#10]
Produce a list of items that have been updated each week for an incremental rsync.
Collaborate with Marie to make sure this is getting generated.

There should be two directories under the `ht_text` rsync point; `delete_log` and `ingest_log`. They should be populated by plaintext lists of htids with the same format as the current id lists. The filenames will be YYYYMMDD and contain a list of all ids removed or added on the named calendar day respectively.

**Comment by aelkiss** [comment]
This is very high priority for HTRC.

**Comment by aelkiss** [comment]
See https://tools.lib.umich.edu/jira/browse/AE-260

**Comment by malakai97** [comment]
1. Is this the problem this is intended to solve:  Using rsync's built-in incremental sync functionality takes too long given the size of the full dataset?
2. What is the history of the interface you describe in your second paragraph?  I'm trying to understand if this is something the HTRC has been using for some time, an idea that has only been brainstormed, or something in the middle of those extremes.

**Comment by aelkiss** [comment]
@rrotter and @skorner could say more about the history of this, but my understanding is:

1) Yes, the incremental sync with the default functionality takes too long
2) It has been promised to HTRC but not yet delivered.

**Comment by malakai97** [comment]
We've decided that how we service HTRC's need does not need to comply with what was promised thus far.  A proposed solution is below; it will contain false positives, but we believe it will address HTRC's needs.

---

Design:

We'd like to have the scheduler produce a list of volumes that are being added and removed.  This is the same code that enqueues the jobs.  This is what we give them:

a "new" list that contains:
* all volumes that have been updated according to feed_audit
* all volumes that have changed according to ht_rights that are still-or-now within ht_text

a "removed" list that contains:
* all volumes that have changed according to ht_rights that are not-or-still-not within ht_text

The "new" and "removed" list above will be supersets of what actually changed.  But `(new+removed).size` is <<<< `everything.size`

Do they only do this to ht_text?  If they also want incrementals for the other subsets then its similar to above, except no feed_audit-driven set.
